.github/workflows/shortest.yml

```diff
@@ -32,8 +32,8 @@ jobs:
       - name: Install dependencies
         run: pnpm install --no-frozen-lockfile
 
-      - name: Install Playwright browsers
-        run: pnpm exec playwright install
+      - name: Install Playwright
+        run: pnpm exec playwright install chromium
```

Comment:
Not clear why this is done? Self-review would be helpful!

Reply:
Since we only use chromium browser, I only install this browser to speed up the execution of workflow.

Reply:
Sounds good. I would always include the why behind changes in the PR description/comments, makes it easier to know / approve an approach.

Reply:
Sure